amf-webapi/shared/src/main/scala/amf/plugins/document/webapi/parser/spec/declaration/OasRefParser.scala

```diff
@@ -128,7 +128,7 @@ class OasRefParser(map: YMap,
       None,
       Some((k: String) => shape.set(LinkableElementModel.TargetId, k)),
       shouldLink = false).withName(declarationNameOrResolvedRef, Annotations()).withSupportsRecursion(true)
-    tmpShape.unresolved(declarationNameOrResolvedRef, e, "warning")(ctx)
```

Comment:
These warnings where adjusted because they where not working and now they do (unresolved was always violation). Don't want to break any validation behaviour.